src/providers/fireworks-ai/imageGenerate.ts

```diff
@@ -0,0 +1,104 @@
+import { FIREWORKS_AI } from '../../globals';
+import { ErrorResponse, ImageGenerateResponse, ProviderConfig } from '../types';
+import { generateInvalidProviderResponseError } from '../utils';
+import {
+  FireworksAIValidationErrorResponse,
+  FireworksAIErrorResponseTransform,
+  FireworksAIErrorResponse,
+} from './chatComplete';
+
+export const FireworksAIImageGenerateConfig: ProviderConfig = {
+  text_prompts: {
```

Comment:
The OpenAI compatible param name is `prompt` which can then be mapped to `text_prompts`

Reply:
done